.github/workflows/release.yaml

```
@@ -13,18 +13,18 @@ jobs:
-      - name: Set up Python 3.10
+        uses: actions/checkout@v4
+      - name: Set up Python 3.12
         id: python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
-          python-version: '3.10'
+          python-version: '3.12'
       - name: Get pip cache dir
         id: pip-cache
         run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)